src/Stratis.Bitcoin.Features.Wallet/Broadcasting/BroadcasterManagerBase.cs

```diff
@@ -33,27 +33,31 @@ public void OnTransactionStateChanged(TransactionBroadcastEntry entry)
             this.TransactionStateChanged?.Invoke(this, entry);
         }
 
+        /// <summary>Transactions to broadcast.</summary>
         private ConcurrentHashSet<TransactionBroadcastEntry> Broadcasts { get; }
 
+        /// <summary>Retrieves a transaction with provided hash from the collection of transactions to broadcast.</summary>
+        /// <param name="transactionHash">Hash of a transaction to retrieve.</param>
```

Comment:
```suggestion
        /// <param name="transactionHash">Hash of the transaction to retrieve.</param>
```

Doing this just to reset the CI...